File: Java/Booking/GetBasketContentResult.java
package LTD_API;

import java.util.Date;
import java.util.List;

public class GetBasketContentResult
{
    private String basketId;
    private Date minExpirationDate;
    private DeliveryInformation deliveryInformation;
    private List<BasketItem> items;
    private DiscountInformation discountInformation;

    public String getBasketId()
    {
        return basketId;
    }
    public void setBasketId(String basketId)
    {
        this.basketId = basketId;
    }

    public Date getMinExpirationDate()
    {
        return minExpirationDate;
    }
    public void setMinExpirationDate(Date minExpirationDate)
    {
        this.minExpirationDate = minExpirationDate;
    }

    public DeliveryInformation getDeliveryInformation()
    {
        return deliveryInformation;
    }
    public void setDeliveryInformation(DeliveryInformation deliveryInformation)
    {
        this.deliveryInformation = deliveryInformation;
    }

    public List<BasketItem> getItems()
    {
        return items;
    }
    public void setItems(List<BasketItem> items)
    {
        this.items = items;
    }

    public DiscountInformation getDiscountInformation()
    {
        return discountInformation;
    }
    public void setDiscountInformation(DiscountInformation discountInformation)
    {
        this.discountInformation = discountInformation;
    }
}

File: PHP/Booking/GetBasketContentResult.php
<?php
namespace LTD_API;

class GetBasketContentResult
{
    private $basketId;
    //date format
    private $minExpirationDate;
    //instance of DeliveryInformation
    private $deliveryInformation;
    //array of BasketItem
    private $items = array();
    //instance of DiscountInformation
    private $discountInformation;
    
    public function getBasketId()
    {
        return $this->basketId;
    }
    public function setBasketId($basketId)
    {
        $this->basketId = $basketId;
    }

    public function getMinExpirationDate()
    {
        return $this->minExpirationDate;
    }
    public function setMinExpirationDate($minExpirationDate)
    {
        $this->minExpirationDate = $minExpirationDate;
    }

    public function getDeliveryInformation()
    {
        return $this->deliveryInformation;
    }
    public function setDeliveryInformation($deliveryInformation)
    {
        $this->deliveryInformation = $deliveryInformation;
    }

    public function getItems()
    {
        return $this->items;
    }
    public function setItems($items)
    {
        $this->items = $items;
    }

    public function getDiscountInformation()
    {
        return $this->discountInformation;
    }
    public function setDiscountInformation($discountInformation)
    {
        $this->discountInformation = $discountInformation;
    }
}

